### ISSUE TITLE
Add a way for Admins to interact with the ban database

### ISSUE THREAD

**Issue by DominickCobb-rs** [Chris7c0/XPMod#1]
Admins currently don't have a well fleshed out way to view or manage bans.

**Comment by Chris7c0** [comment]
I think its probably better to keep this self contained inside of XPMod in game. This way admins wont have to setup and learn of outside software.  Maybe we can ask the Admin to copy paste a steamid as a parameter in console/chat command to unban someone?  We could also have a fuzzy finder that looks for names and spits out steam ids for matches.  Actions taken should definitely be logged into the mysql database somehow.

**Comment by DominickCobb-rs** [comment]
I think for lightweight work for ban management in game it could be done via the console and console commands given to admins. 

Another idea is an unlisted admin server with a DDNS name of something like "admin.xpmod.net" that allows admins to interface without interfering with a live game. Downside to this is not having steamids they need unless supplied or stored elsewhere

**Comment by DominickCobb-rs** [comment]
I think it is a good idea to keep it in game for security purposes. Exposing the db to the internet is a bad idea as stated by ChrisP.

**Comment by Chris7c0** [comment]
yeah maybe down the road it will be good to have an easy to use web interface, but for now self containing it inside of in game seems like the quickest way to get admins some more functionality.
